src/components/courses/AdminPage.tsx

```
@@ -56,75 +55,120 @@ const AdminPage = () => {
   }, [reloadData]);
 
-  useEffect(() => {
-    fetchCourses();
-  }, []);
-
-  const handleAddCourse = (formData: FormData) => {
+  const handleAddCourse = (formData: any) => {
     try {
       let user: any;
       let userJson = localStorage.getItem("user");
       if (userJson) user = JSON.parse(userJson);
```

Biome

[error] This let declares a variable that is only assigned once.

'userJson' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)